src/utilities/renderer-helper.ts

```diff
@@ -410,6 +415,30 @@ class RendererHelper {
             throw new WrappedError(errorMessagePrefix, e);
         }
     }
+
+    isPathAccessibleToRenderer = async ({
```

Comment:
Is this used in this PR? I thought I saw it, but I don't anymore.

Reply:
Yup here: https://github.com/rederly/backend/pull/298/files?diff=split&w=1#diff-bd3a5d3bcddc58f765ae924aeb9f25446d6f54deafece77a322f95fb9f32ed24R3078